diff --git a/node_modules/vite/dist/node/chunks/dep-c1a9de64.js b/node_modules/vite/dist/node/chunks/dep-c1a9de64.js
index 9f04e5f..eb05504 100644
--- a/node_modules/vite/dist/node/chunks/dep-c1a9de64.js
+++ b/node_modules/vite/dist/node/chunks/dep-c1a9de64.js
@@ -51192,8 +51192,10 @@ function esbuildScanPlugin(config, container, depImports, missing, entries) {
                     }
                     else {
                         // linked package, keep crawling
+                        const namespace = htmlTypesRE.test(resolved) ? 'html' : undefined
                         return {
-                            path: path__default.resolve(resolved)
+                            path: path__default.resolve(resolved),
+                            namespace
                         };
                     }
                 }
diff --git a/node_modules/vite/src/node/optimizer/scan.ts b/node_modules/vite/src/node/optimizer/scan.ts
index 9193116..ed9ce51 100644
--- a/node_modules/vite/src/node/optimizer/scan.ts
+++ b/node_modules/vite/src/node/optimizer/scan.ts
@@ -283,8 +283,10 @@ function esbuildScanPlugin(
               return externalUnlessEntry({ path: id })
             } else {
               // linked package, keep crawling
+              const namespace = htmlTypesRE.test(resolved) ? 'html' : undefined
               return {
-                path: path.resolve(resolved)
+                path: path.resolve(resolved),
+                namespace
               }
             }
           } else {
